Sources/StreamChat/APIClient/Endpoints/Payloads/CurrentUserPayloads_Tests.swift

```diff
@@ -35,7 +35,7 @@ class CurrentUserPayload_Tests: XCTestCase {
         XCTAssertEqual(payload.mutedChannels[0].user.id, "broken-waterfall-5")
         XCTAssertEqual(payload.mutedChannels[0].mutedChannel.cid.rawValue, "messaging:B1DFF9C5-E6A6-4BFA-9375-DC5E8C6852FF")
         XCTAssertEqual(payload.mutedChannels[0].createdAt, "2021-03-22T10:23:52.516225Z".toDate())
-        XCTAssertEqual(payload.mutedChannels[0].updatedAt, "2021-04-22T10:23:52.516225Z".toDate())
+        XCTAssertEqual(payload.mutedChannels[0].updatedAt, "2021-03-22T10:23:52.516225Z".toDate())
```

Comment:
This looks suspicious

Reply:
But it is not. It was copy paste error in the `Channel` object in `muted_channel` vs the `muted_channel` object which contains both `user` and `channel`. This is fix, this should already fail in https://github.com/GetStream/stream-chat-swift/pull/1011   


```
"muted_channel" : {
.... 
"channel":{ "createdAt" : RightDate }
"createdAt" : WrongDateUsedAtTests
}```